code/data_structures/src/tree/tree/trie/trie.c

```diff
@@ -7,63 +7,68 @@
 #define ALPHABET_SIZE (26)
 #define INDEX(c) ((int)c - (int)'a')
 #define FREE(p) \
-    free(p);    \
-    p = NULL;
- 
+free(p);    \
+p = NULL;
+
 typedef struct trie_node trie_node_t;
- 
-struct trie_node
+
+struct 
+trie_node
 {
     int value; // non zero if leaf
     trie_node_t *children[ALPHABET_SIZE];
 };
- 
+
 typedef struct trie trie_t;
- 
-struct trie
+
+struct 
+trie
 {
     trie_node_t *root;
     int count;
 };
- 
-trie_node_t *getNode(void)
+
+trie_node_t* 
+getNode(void)
 {
     trie_node_t *pNode = NULL;
     pNode = (trie_node_t *)malloc(sizeof(trie_node_t));
- 
+    
     if( pNode )
     {
         int i;
         pNode->value   = 0;
- 
+        
```

Comment:
Is whitespace on blank lines like this actually wanted?

Reply:
@polyzen I don't know what the standards are in such cases, but my editor(Kate) adds whitespaces in blank lines when aligning the code. I couldn't find anything about it on the Internet.

Reply:
It's subjective, not objective, but trailing whitespace is more often frowned upon than not. Even more frowned upon when a patch clobbers history of lines like this for no reason other than whitespace changes.